wallet/how-to/display/tokens.md

```diff
@@ -19,11 +19,14 @@ You can improve the security and experience of displaying your
 `wallet_watchAsset` provides a friendly interface that prompts users to register tokens to their
 MetaMask wallet, without having to interact with contract addresses.
 
-:::tip Displaying NFTs
-With `wallet_watchAsset`, you can prompt users to add their NFTs even when they have NFT
-autodetection disabled.
-Moreover, NFT autodetection only detects NFTs on Ethereum Mainnet.
-With `wallet_watchAsset`, users can add NFTs from other networks.
+:::tip
+With `wallet_watchAsset`, you can prompt users to add their ERC-20 tokens and NFTs even when they
```

Comment:
I think this a passive construction. Maybe...
```suggestion
"Use `wallet_watchAsse`t to prompt users to add their ERC-20 tokens and NFTs even when token autodetection or NFT autodetection is disabled."
```

Reply:
The intention of this tip is to expand on the benefits of using `wallet_watchAsset`. It's not necessarily an instruction, hence the wording.